### ISSUE TITLE
Stand alone version?

### ISSUE THREAD

**Issue by vgavrilovikj** [Kwaadpepper/laravel-responsivefilemanager#17]
How can I use it as a stand-alone? I have followed the original documentation on their site (https://responsivefilemanager.com/index.php#documentation-section) however when I implemented it when I click on the select button and choose an image, nothing happens... the function responsive_filemanager_callback() is not even called..

Also when I click on select button on the top, after selecting 2 images, I get this error in console.log 
![image](https://user-images.githubusercontent.com/29601489/99851446-7a710400-2b7f-11eb-9bd0-7c39a8ed58e4.png)

Here's my code:
```
<input type="text" class="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline" id="fieldID" placeholder="Images" wire:model="images" value="">
                            <img style="width:20%;" class="image-preview" src="">
                            <a href="filemanager/dialog.php?type=1&relative_url=1&field_id=fieldID&relative_url=1&multiple=1&akey=@filemanager_get_key()" class="btn iframe-btn" type="button">Select</a>

<script type="text/javascript">
    $('.iframe-btn').fancybox({  
        'width'    : 600,
        'height'  : 600,
        'type'    : 'iframe',
        'autoScale'      : false
    });
    
    function responsive_filemanager_callback(field_id){
        console.log(field_id);
        var url=jQuery('#'+field_id).val();
        $(".image-preview").attr('src','http://localhost/filemanager/plugins/source/'+ url);
        alert('update '+ field_id +" with "+ url);
    }
</script>
```

**Comment by vgavrilovikj** [comment]
@Kwaadpepper can you please take a look at this? I need this urgently.. Thank you!

**Comment by Kwaadpepper** [comment]
You are missing jquery. Include jquery script tag in your HTML.
By the way this project is meant to be used with laravel...

**Comment by vgavrilovikj** [comment]
Yes, jQuery is already included in the HTML, in the master layout.. and I am using Laravel - the code snippet is not the whole site.. only where I call it.

**Comment by Kwaadpepper** [comment]
Just replace jQuery with $

**Comment by vgavrilovikj** [comment]
In the include.js? @Kwaadpepper

**Comment by Kwaadpepper** [comment]
In your snippet

**Comment by Kwaadpepper** [comment]
Maybe try another jQuery version. Sorry cant do more for now.

**Comment by vgavrilovikj** [comment]
@Kwaadpepper  I tried that one and it fixed the issue with jQuery. I need this working, otherwise, I am not able to upload files from the file manager.. However, now there is another issue (Cannot read property 'document' of null):
![image](https://user-images.githubusercontent.com/29601489/99859021-240bc180-2b8f-11eb-83f7-3a56d2d5c27a.png)

**Comment by Kwaadpepper** [comment]
It is blocked by your browser. Maybe CSP policies? You may need to check server side.

**Comment by vgavrilovikj** [comment]
@Kwaadpepper  Hmm.. what do you mean CSP policies? I have not made any customization to any CSP policies..
When I put breakpoints in the include.js file this is where it breaks
![image](https://user-images.githubusercontent.com/29601489/99859342-13a81680-2b90-11eb-9ed3-c09016f081e2.png)
Because in the apply = function (), t = null, and on line 1102 it is trying to access t.document and that is why it throws that error..

**Comment by Kwaadpepper** [comment]
This code hasnt change for years. It is not written by me. Its goal IS.to access the iframe data. The browser is blocking it for some reason.

**Comment by vgavrilovikj** [comment]
@Kwaadpepper okay, that issue was fixed - it was because I had "&popup=1" in the URL, I removed it and it works just fine. However back to the first issue = Stand alone version..?

This is my input field and the select button:
```
<input type="text" id="fieldID" value="" class="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline" placeholder="Images">
<a href="@filemanager_get_resource(dialog.php)?type=0&relative_url=1&field_id=fieldID&relative_url=1&multiple=1&akey=@filemanager_get_key()" class="btn iframe-btn" type="button">Select</a>
```

When I click on "Select" the file manager opens I select 2 images then again click on "Select" on the top however the input filed with id "fieldID" is not populated with the selected images.. it stays empty..

Do you have any code where it shows how it works as stand-alone? Thanks!

**Comment by Kwaadpepper** [comment]
Sorry but I only manage to adapt a bit RFM to laravel. This functionnality was meant to manage files, not to be used within an input. No data is returned as far as I Can remember.

**Comment by Kwaadpepper** [comment]
You may need to follow RFM Doc https://responsivefilemanager.com/index.php#documentation-section

**Comment by Kwaadpepper** [comment]
The *CALLBACK FUNCTION CALL* part

**Comment by vgavrilovikj** [comment]
@Kwaadpepper ah I see.. noo - the functionality is for both, file manager and file picker - THat's why on their site under "Demo" it says "Select Only Images (multiple selection)":
![image](https://user-images.githubusercontent.com/29601489/99860418-50c1d800-2b93-11eb-90b9-e1916c82a871.png)

That is the point of the function "function responsive_filemanager_callback (field_id)" however it doesn't work.. because it is never fired and I cannot figure out why..

I have followed their Docs, this is how my function looks like, but there is nothing in the console, nor any alerts..

```
function responsive_filemanager_callback(field_id){
	alert("HELLOOOOO");
        console.log(field_id);
        var url= $('#'+field_id).val();
        $(".image-preview").attr('src','http://localhost/filemanager/plugins/source/'+ url);
        alert('update '+ field_id);
}
```

**Comment by vgavrilovikj** [comment]
Wow.. figured it out.... you need to put "&field_id=" at the end of the href and it works just fine.. Sorry again, not your fault! Thank you so much! @Kwaadpepper

**Comment by vgavrilovikj** [comment]
Although it still doesn't call the function, but you get the images' names in the input field..
